index.rst

```diff
@@ -1,57 +1,343 @@
+
+  - Regular operations routines should be easily accessible;
+  - Must include a highly customizable scripting capability must be available and able to run user-written code;
+  - An interactive mode, equivalent to a user sandbox space, must be available to allow development of routines on the fly as well as debugging;
+  - Capability to incorporate user-developed features;
+
+In the standard operational model used to perform the survey, the observatory will be largely driven by the observatory Scheduler.
+The Scheduler designates a target and a specially formatted script containing a series of operations to be performed (such as slewing the telescope and taking an image).
+The execution of the script handled by the control system while the user is able to observe (or interrupt) the progress via a graphical interface.
+Operators will be much more involved during the start-up and shutdown phases, as well as during afternoon calibrations.
+In the majority of cases, these routine operations are expected to be executed by the operator from a high-level interface of the observatory control software (as discussed in following sections).
+
+During integration, commissioning, and engineering time, there will be situations where non-standard, customized operations will need to be executed.
+These operations will often be exercised by specialists to a specific subsystem (e.g. M1M3) and require a knowledge base that differs from that of an observing specialist.
+The breadth of the observatory control system allows users to perform both standard routines and customized tasks via three main interactive components:
+
+    - :ref:`LSST Observing Visualization Environment (LOVE)<LOVE>`
+    - :ref:`ScriptQueue <Script-Queue>`
+    - :ref:`Jupyter notebooks <Jupyter-notebooks>`
+
+This document begins with a high-level overview of the three components and explores how users can take advantage of their features to accomplish different types of tasks.
+Then, we define three main actors and outline their roles and assumed levels of knowledge regarding when performing specific tasks.
+These actors and roles are used to assist in defining the usability expectations of the components and control system in general as well as the underlying assumptions of the knowledge required for each role.
+In reality, the people's roles will be less discrete and there will be significant overlap between these roles.
+The identified actors are:
+
+    1. Observing Specialists
+    2. Commissioning Scientists
+    3. Software Developers
+
+Finally, we present concrete examples of interactions that the actors will perform with the OCS, linking to procedures on how to do them using the multiple components.
+These examples explore how users best take advantage of the component's features.
+They also include cases of how *not* to use a component due limitations or inefficiencies in various areas.
```

Comment:
I think "not" should be in **bold**  rather than in *italic*. 

Also, add "to" between "due limitations"?

```
They also include cases of how **not** to use a component due to limitations or inefficiencies in various areas.
```